stories/Defer.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+export interface IDefer<T> {
+  promise: Promise<T>;
+  children: (data: T) => React.ReactNode;
+}
+
+interface IDeferState {
+  data: any;
```

Comment:
Nitpick: Maybe this could be a `never` so we're sure nobody will ever touch it from here and just pass it through.